tests/fixtures.py

```diff
@@ -32,16 +31,21 @@ def load_config():
 
 
 @pytest.fixture(scope='session')
-def set_namespace(request, load_config):
+def session_id():
+    return random_id(length=5)
```

Comment:
wouldn't that return a different id every time called?

Reply:
if it happens only as a fixture and it's scoped to a session then only once a session right?

Reply:
Yes. But its scope is session, so it is called only once per session